android/feature-login/src/main/java/io/newm/feature/login/screen/createaccount/CreateAccountUiState.kt

```diff
@@ -0,0 +1,14 @@
+package io.newm.feature.login.screen.createaccount
+
+import com.slack.circuit.runtime.CircuitUiState
+import io.newm.feature.login.screen.TextFieldState
+import io.newm.feature.login.screen.createaccount.signupform.SignupFormUiEvent
+
+sealed interface CreateAccountUiState : CircuitUiState {
+    data class SignupForm(
```

Comment:
This looks great, curious with this approach will we be able to reuse states like Loading, Error or will those states have to exist within CreateAccountUiState seal class?